compiler/vm/vmgen.nim

```diff
@@ -1697,17 +1733,17 @@ proc genMagic(c: var TCtx; n: PNode; dest: var TDest; m: TMagic) =
       # do not call clearDest(n, dest) here as getAst has a meta-type as such
       # produces a value
     else:
-      fail(n.info, vmGenDiagBadExpandToAst,
-        str = "expandToAst requires a call expression")
+      fail(n.info, vmGenDiagBadExpandToAstCallExprRequired)
+        # str = "expandToAst requires a call expression")
 
   of mSizeOf:
-    fail(n.info, vmGenDiagMissingImportcCompleteStruct, str = "sizeof")
+    fail(n.info, vmGenDiagMissingImportcCompleteStruct, m)
```

Comment:
You can combine the `mSizeOf`, `mAlignOf`, and `mOffsetOf` branches into a single one now.

Reply:
Thanks, done.